### PR TITLE
Add UI support for alternate models

### PR DESCRIPTION
**Pull request by QuiteQuiet**
Depends on https://github.com/Zarel/Pokemon-Showdown-Client/pull/702 and https://github.com/Zarel/Pokemon-Showdown/pull/2539

Looks like: http://imgur.com/a/yFd0C

### CODE REVIEW

**Comment by Slayer95** [comment]
Maybe make the sprite/model itself clickable?

**Comment by QuiteQuiet** [comment]
My only concern with doing that is how do you show to users that the model is clickable without doing something with the current layout of the Pokemon card? Unless it's obvious, people will miss it.

**Comment by Slayer95** [comment]
Well, the grand majority of users almost never click the ``Details`` menu.
I guess now that we don't autofix shinyness they do a bit more often?

Giving it ``cursor:pointer`` is enough to convey the clickability IMO.
We could also have both UIs...

**Comment by Zarel** [comment]
I would suggest both UIs.

**Comment by QuiteQuiet** [comment]
Added clickable Pokemon icons, but I still don't think it's obvious enough that the icon is clickable even with something like `cursor: pointer` but I don't have any good ideas how to show it...

**Comment by Slayer95** [comment]
If you are willing to, you could add a [caret-down](https://upload.wikimedia.org/wikipedia/commons/thumb/9/9a/Caret_down_font_awesome.svg/512px-Caret_down_font_awesome.svg.png) in the bottom-right corner of the icon, I guess

**Comment by QuiteQuiet** [comment]
Something like this (can be made larger)?

**Comment by Zarel** [comment]
You might want an outline around it, and probably pick a color other than pure black. Maybe something like blue-gray with a white outline?

**Comment by QuiteQuiet** [comment]
Bright borders were really hard to get visible, so I tried a darker version too:

Or something entirely different: (I kind of like this version, to be honest)

**Comment by Zarel** [comment]
That wasn't actually what I meant but that looks pretty good, too.

**Comment by QuiteQuiet** [comment]
Oh you meant a border around the entire sprite?

Is this fine, or do you want to look at the border around the sprite too? I don't think I can make the border around the sprite look good, when it's going to be broken by everything around it...

**Comment by Zarel** [comment]
No, this is better than what I had in mind. Which was a border around the arrow, not a box around the arrow.

**Comment by Zarel** [comment]
although adding a box-shadow anyway might still look good.

**Comment by QuiteQuiet** [comment]
Updated for:

I'm still not sure this is the best colour to use here, to be fair...

File: style/client.css [hunk 1]
```diff
@@ -2906,6 +2906,31 @@ a.ilink.yours {
...
+	padding-top: 2px;
+	width: 12px;
+	height: 14px;
+	position: relative;
+	left: 90px;
+	border: 1px solid #569;
+	border-radius: 2px;
+	color: #569;
+	-webkit-box-shadow: 0 1px 2px rgba(0,0,0,.2);
+	-moz-box-shadow: 0 1px 2px rgba(0,0,0,.2);
```
Comment (Slayer95) [inline style/client.css]: Do we still need these? @Zarel

I see we are seldom using the prefixed versions for ``box-shadow``

Comment (Zarel) [inline style/client.css]: I wouldn't use prefixed versions.

**Comment by QuiteQuiet** [comment]
Updated.

**Pull request merged**